src/modules/auth/containers/LoginScreen.js

```diff
@@ -59,31 +53,6 @@ function LoginScreen(props) {
 		}
 	}
 
-	function useWindowSize() {
```

Comment:
the `useWindowSize` is only being used to measure and then re-create background when screen size changes (same was in RegisterScreen) 

it's now part of the `generatePenrose` function